xikolo-ios/ViewControllers/News/NewsArticleViewController.swift

```diff
@@ -12,14 +12,21 @@ class NewsArticleViewController : UIViewController {
 
     @IBOutlet weak var titleView: UILabel!
     @IBOutlet weak var textView: UITextView!
+    @IBOutlet weak var dateView: UILabel!
 
     var newsArticle: NewsArticle!
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        let dateFormatter = NSDateFormatter()
+        dateFormatter.dateStyle = .MediumStyle
+        dateFormatter.timeStyle = .NoStyle
```

Comment:
I don't know how frequent new articles come in, but I think it would only matter in a specific course.